app/models/sighting.rb

```diff
@@ -4,4 +4,13 @@ class Sighting < ApplicationRecord
   validates :last_seen_at, presence: true
   validates :date_time, presence: true
   mount_uploader :image, ImageUploader
+
+  extend FriendlyId
+  friendly_id :slug_candidates, use: [:slugged, :history]
+  def slug_candidates
+    [
+      :pet_type,
```

Comment:
Would consider removing this line, as the pet_type would be populated pretty quick.